sphinx_js/doclets.py

```diff
@@ -140,6 +143,39 @@ def analyzer_for(language):
         raise SphinxError('Unsupported value of js_language in config: %s' % language)
 
 
+def process_see(doclets):
+    """Process @see tags for all doclets."""
+    for doclet in doclets:
+        if 'see' in doclet:
+            see = doclet['see']
+            process_see_list(see)
+
+
+def process_see_list(see):
+    """Process a list of @see tag values.
+
+    Leaves any value with a space as freeform text.
+    Translates other values into RST crossreferences.
+    """
+    for i, item in enumerate(see):
+        if ' ' in item:
```

Comment:
I think your heuristic is fine for now. Of course, you could implement the very test you described elsewhere, trying to resolve the path and falling back to freeform on failure. All you have to do is look it up in _sphinxjs_doclets_by_path. Grep for that in renderers.py for an example.